platform/mellanox/docker-syncd-mlnx/Dockerfile.j2

```diff
@@ -70,4 +71,10 @@ RUN mkdir -p /etc/supervisor/conf.d/
 RUN sonic-cfggen -a "{\"ENABLE_ASAN\":\"{{ENABLE_ASAN}}\"}" -t /usr/share/sonic/templates/supervisord.conf.j2 > /etc/supervisor/conf.d/supervisord.conf
 RUN rm -f /usr/share/sonic/templates/supervisord.conf.j2
 
+RUN mkdir -p /usr/bin/ecmp_calculator
+COPY ["ecmp_calculator/ecmp_calc_sdk.py", "/usr/bin/ecmp_calculator"]
+COPY ["ecmp_calculator/ecmp_calc.py", "/usr/bin/ecmp_calculator"]
```

Comment:
Apart from ecmp_calc.py other files are not binaries but libraries.Can we package them accordingly?

Reply:
Done.